internal/host/validations_test.go

```diff
@@ -414,6 +414,23 @@ var _ = Describe("Validations test", func() {
 			Expect(validationMessage).To(Equal("TPM version could not be found, make sure TPM is enabled in host's BIOS"))
 		})
 
+		It("day2 host - disk encryption is not available", func() {
+			createDay2Cluster()
```

Comment:
looking at createDay2Cluster(), but I suspect it does not create a day2 function, can you please double-check? that might also explain why this is passing despite my comment above.

This function calls GenerateTestCluster(), and does not change anything with cluster Kind
https://github.com/openshift/assisted-service/blob/b2907295956224410f3572858eb2e72a84586c0f/internal/host/validations_test.go#L127-L131

GenerateTestCluster() is using the day1 cluster kind ClusterKindCluster
https://github.com/openshift/assisted-service/blob/b2907295956224410f3572858eb2e72a84586c0f/internal/host/hostutil/test_utils.go#L25-L38